controller_manager/controller_manager/spawner.py

```diff
@@ -162,14 +122,6 @@ def main(args=None):
         action="store_true",
         required=False,
     )
-    parser.add_argument(
-        "--fallback_controllers",
-        help="Fallback controllers list are activated as a fallback strategy when the"
-        " spawned controllers fail. When the argument is provided, it takes precedence over"
-        " the fallback_controllers list in the param file",
-        default=None,
-        nargs="+",
-    )
```

Comment:
This is supposed to be a refactor. Why are we removing things without deprecation? Should go at least into a separate PR so it shows in the changelogs

Reply:
You are right! Sorry my bad. 
The reason I removed it is that the feature of fallback controllers is not complete, so it's better to remove it than add a deprecation note at this point. 

What do you think?

Reply:
I agree with you @saikishor.

This feature is not know very well as it is new and probably nobody exept yourself is using it right now.

Reply:
thank you, let's roll with it then!